config.json

```diff
@@ -517,6 +523,12 @@
             "ContainerName": "default"
           }
         },
+        "nodeReuse":{
```

Comment:
Do we have any good way to pass default parameters to a tool definition instead of a command? Somewhere https://github.com/maririos/corefx/blob/9f496775b06eb8f9ff1d7761eec8d3198145c8fb/config.json#L521?